.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature Request
+about: Help us make Upbound more useful
+labels: enhancement, needs-triage, needs-size-label, needs-epic-link
```

Comment:
I think `needs-size-label` and `needs-epic-link` are redundant. I think triaging an issue should mean swagging it, and parenting it with the right epic.

Reply:
I think distinct labels work as better specific prompts vs a generic “triage” process while we don’t have automation. I don’t feel strongly though.

@grantgumina I wonder if it makes sense to address this by renaming `needs-triage` to something more specific and actionable. Not sure on the wording but I'm thinking something along the lines of "needs to be assessed for a sprint". Maybe `needs-priority`?

Reply:
I went with `needs-project`.